packages/environment-check/src/output/zip.ts

```diff
@@ -20,6 +20,27 @@ function byteNumberToSizeString(byteNumber: number): string {
     return `${parseFloat((byteNumber / Math.pow(1024, i)).toFixed(2))} ${units[i]}`;
 }
 
+/**
+ * Convert results to JSON and removes credentials from (stored system) endpoints.
+ *
+ * @param results - environment check results
+ * @returns json string
+ */
+function convertResultsToJson(results: EnvironmentCheckResult): string {
```

Comment:
@cianmSAP, any chance we remove the credential at the time we collect the data so they are never part of the results?

Reply:
Made updates to never return credentials from the exposed functions